docs/reference/custom resources/key-management-providers.md

```diff
@@ -1,11 +1,11 @@
-Key Management Provider
----
+## Key Management Provider
```

Comment:
nit: should this be a top level header element?

Reply:
Looking at the other custom resource docs, yes I believe it does. I went ahead and changed that to be an H1.